reference/tax_provider.yml

```diff
@@ -659,11 +659,14 @@ components:
           $ref: '#/components/schemas/TaxClass'
         tax_exempt:
           type: boolean
-          description: Flag whether or not this item is always tax exempt. Gift Certificate purchases for example. Tax Exempt items should be included in the Request for auditing purposes.
+          description: Flag whether or not this item is always tax exempt. For example, for gift certificate purchases and order-level refunds. Tax exempt items are included in the request for auditing purposes.
           default: 'false'
         type:
           type: string
-          description: 'The type of line item this request represents. This will depend on the items position in the request hirearchy, for example a collection of items (Which may or may not also have wrapping attached) are contained within the document request, and separately each document request has a single shipping and a single handling line item (to capture these values).'
+          description: |-
+            The type of line item this request represents. This will depend on the items position in the request hirearchy, for example a collection of items (Which may or may not also have wrapping attached) are contained within the document request, and separately each document request has a single shipping and a single handling line item (to capture these values).
```

Comment:
🍹 This sentence (Which I know was existing) feels rather run-on.

Reply:
Thanks for the suggestion, updated in this [commit](https://github.com/bigcommerce/api-specs/pull/309/commits/f1494fc63c72e5c63dd648d7fb7e30f02fc930ad).